### PR TITLE
Add redis_host options

### PR DESCRIPTION
**Pull request by calston**

### CODE REVIEW

**Comment by calston** [comment]
Requires https://github.com/praekeltfoundation/puppet-xylem/pull/6

File: metadata.json [hunk 1]
```diff
@@ -9,43 +9,43 @@
   "issues_url": "https://github.com/praekeltfoundation/puppet-seed_stack/issues",
   "dependencies": [
     {
-      "name": "praekeltfoundation/consular",
+      "name": "praekeltfoundation-consular",
```
Comment (JayH5) [inline metadata.json:12]: What's the reason for this?

File: manifests/xylem.pp [hunk 2]
```diff
@@ -7,6 +7,7 @@
   $gluster_mounts,
   $gluster_replica = undef,
   $gluster_stripe  = undef,
+  $redis_host      = '127.0.0.1',
```
Comment (JayH5) [inline manifests/xylem.pp]: Would prefer not to default to an explicit IP address. Can use `$::ipaddress_lo` rather.

File: metadata.json (hunk 1 above)
Comment (calston) [inline metadata.json:12]: The / separator is deprecated and librarian is throwing warnings

**Comment by jerith** [comment]
:+1:

**Pull request merged**